packages/jaeger-ui/src/components/QualityMetrics/Header.tsx

```diff
@@ -24,25 +24,25 @@ type TProps = {
   lookback: number;
   service?: string;
   services?: string[] | null;
-  setLookback: (lookback: number | string | undefined) => void;
+  setLookback: (lookback: number | null) => void;
```

Comment:
Can you elaborate why you're changing undefined to null?

Reply:
In antd v4.23.1, they added the `null` type instead of `undefined`.

https://4x.ant.design/changelog#4.23.1
https://github.com/ant-design/ant-design/pull/37421